pkg/ccl/importccl/import_stmt.go

```diff
@@ -1670,12 +1670,45 @@ func (r *importResumer) dropTables(
 		}
 	}
 
+	var offlineUntil hlc.Timestamp
 	for i := range empty {
-		if err := gcjob.ClearTableData(ctx, execCfg.DB, execCfg.DistSender, execCfg.Codec, empty[i]); err != nil {
-			return errors.Wrapf(err, "clearing data for table %d", empty[i].ID)
+		tableSpan := empty[i].TableSpan(execCfg.Codec)
+		var err error
+
+		for d := time.Millisecond * 25; d < time.Minute; d *= 2 {
+			offlineUntil.Forward(execCfg.Clock.Now().Add(d.Nanoseconds(), 0))
+			log.VEventf(ctx, 2, "ClearRange %s - %s (deadline +%s)", tableSpan.Key, tableSpan.EndKey, d)
+			var b kv.Batch
+			b.AddRawRequest(&roachpb.ClearRangeRequest{
+				RequestHeader: roachpb.RequestHeader{
+					Key:    tableSpan.Key,
+					EndKey: tableSpan.EndKey,
+				},
+				Deadline: offlineUntil,
+			})
+			err = execCfg.DB.Run(ctx, &b)
+			if err == nil {
+				break
+			}
+			if strings.Contains(err.Error(), "ClearRange has deadline") {
```

Comment:
wouldn't it need to be a roachpb-defined error?

I went back and forth a little with Nathan on the original PR about this (I was using ConditionFailed since I argued the deadline was a condition) but he and other KV folks didn't like it, and said to just match the string.

Reply:
https://github.com/cockroachdb/cockroach/pull/52739#issuecomment-673653417

Reply:
Hmm, I thought that `roachpb.Error`s encodes the original go error so that it can be retrieved.

I was thinking that something like https://github.com/pbardea/cockroach/commit/a3dc088bb54f6ee5192e93bd1197bbcafa02ca61 should work. (`make test PKG=./pkg/ccl/importccl TESTS=TestImportIntoCSV` hits the `error detected` panic on that commit)